### PR TITLE
Move Tofu resource creation to hooks dir

### PR DESCRIPTION
**Pull request by eshulman2**
moved tofu to hooks dir and removed unnecessary vars and plays

As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

File: hooks/playbooks/run_tofu.yml [hunk 1]
```diff
@@ -19,6 +19,21 @@
...
+    - name: Create openstack config dir
+      ansible.builtin.file:
+        path: /home/zuul/.config/openstack/
+        owner: zuul
+        group: zuul
+        mode: '0744'
+        state: directory
+    - name: Fetch cloud congig to host
+      kubernetes.core.k8s_cp:
+        kubeconfig: "{{ cifmw_tufo_kubeconfig }}"
```
Comment (pablintino) [inline hooks/playbooks/run_tofu.yml:31]: tufo? typo

File: hooks/playbooks/run_tofu.yml [hunk 2]
```diff
@@ -19,6 +19,21 @@
   gather_facts: true
   run_once: true
   tasks:
+    - name: Create openstack config dir
+      ansible.builtin.file:
+        path: /home/zuul/.config/openstack/
```
Comment (pablintino) [inline hooks/playbooks/run_tofu.yml:24]: Can we use the ansible_user_dir here?

Comment (cjeanner) [reply hooks/playbooks/run_tofu.yml:24]: depends where the role is called - nested ansible, delegate_to, or... ?

File: hooks/playbooks/run_tofu.yml (hunk 1 above)
Comment (eshulman2) [reply hooks/playbooks/run_tofu.yml:31]: yes :woman_facepalming:

**Review by raukadah — APPROVED** [review]

**Pull request merged**